test/README.md

```diff
@@ -70,6 +70,14 @@ the ssh agent to provide the correct credentials.
 `API_PORT` should be set when connections are performed through a
 forwarded port.
 
+To ensure the router smoke test works properly on a Microshift host,
+you need to install the `avahi-resolve-host-name` command, which is
+essential for hostname resolution. You can install it using the following command:
+
+```
+sudo dnf install -y "*/bin/avahi-resolve-host-name"
```

Comment:
```suggestion
sudo dnf install -y avahi-tools
```

Reply:
I double-checked and we have this package installed [here](https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/configure-vm.sh#L139). I do not mind adding another "reminder" in the test docs.